.devcontainer/setup.sh

```diff
@@ -2,11 +2,9 @@
 
 BAZELRC_FILE=~/.bazelrc bazel/setup_clang.sh /opt/llvm
 
-# TODO(phlax): use user.bazelrc
-# Use generated toolchain config because we know the base container is the one we're using in RBE.
-# Not using libc++ here because clangd will raise some tidy issue in libc++ header as of version 9.
-echo "build --config=rbe-toolchain-clang-libc++" >> ~/.bazelrc
-echo "build ${BAZEL_BUILD_EXTRA_OPTIONS}" | tee -a ~/.bazelrc
+if [[ -n "${BAZEL_BUILD_EXTRA_OPTIONS}" ]]; then
+    echo "build ${BAZEL_BUILD_EXTRA_OPTIONS}" | tee -a ~/.bazelrc
```

Comment:
im wondering if this should be added to user.bazelrc instead - also wondering if there is a risk of the flags repeating

i dont use this, but the change looks otherwise sensible to me

Reply:
yeah to be honest this could be deleted and users can simply put this either in their user.bazelrc or the global ~/.bazelrc by hand - I see no need for this setup.sh to do this on behalf of them

Reply:
sgtm - i suspect there are few if any vscode/rbe users - cc @envoyproxy/maintainers